Epsilon.Host.Frontend/src/components/TopNavigation.vue

```diff
@@ -3,13 +3,13 @@
         <img alt="logo" class="header-logo" src="../assets/logo.png" />
         <Row>
             <Col :cols="9">
-                <div class="d-flex">
-                    <SearchBox
-                        v-model="store.state.currentUser"
-                        :items="store.state.users"
-                        :limit="5"
-                        placeholder="Student" />
-                </div>
+                <!--                <div class="d-flex">-->
+                <!--                    <SearchBox-->
+                <!--                        v-model="store.state.currentUser"-->
+                <!--                        :items="store.state.users"-->
+                <!--                        :limit="5"-->
+                <!--                        placeholder="Student" />-->
+                <!--                </div>-->
```

Comment:
I saw this was commented due to the fact that it isn't being used yet. Maybe add a TODO above the comment to indicate the situation. This will make it clearer for the other team members.